recipes/leveldb/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,20 @@
+#include "leveldb/db.h"
+#include <iostream>
+
+int main()
+{
+  leveldb::DB *db;
+  leveldb::Options options;
+  options.create_if_missing = true;
+  leveldb::Status status = leveldb::DB::Open(options, "/tmp/testdb", &db);
```

Comment:
```suggestion
  leveldb::Status status = leveldb::DB::Open(options, "testdb", &db);
```

What happen if you use locally?